index.html

```diff
@@ -36,26 +42,24 @@ <h4 class="headline">LET’S CONNECT</h4>
       <li><a href="#"><img src="images/Vector(1).svg" alt="gitbubicon" /></a></li>
       <li><a href="#"><img src="images/Vector.svg" alt="thumb" /></a></li>
     </ul>
-  </section>
-
+  </div>
+</section>
 
-
-
-
-
-  
-  <section>
-    <div class="cards-container">
-      <ul class="cards-list flex-center-culomn">
-        <li class="card-item flex-center-culomn">
-          <img class="card-image" src="images/SnapshootPortfolio.jpg" alt="Portfolio Portfolio">
+<section>
+  <div class="cards-container">
```

Comment:
### OPTIONAL 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_29](https://user-images.githubusercontent.com/69638013/161253687-1d6108a6-3593-4ce8-bcbf-64a527e62536.png)  |  ![Screenshot_28](https://user-images.githubusercontent.com/69638013/161253625-1ae0aba1-f268-4bdf-b249-d791a7016571.png)

- [ ] Kindly adjust the left padding of the card list items, it should look like the Figma design, please do this for all card list items 👍🏿